netlify.toml

```diff
@@ -0,0 +1,42 @@
+[build]
```

Comment:
We should remove this since we don't use Netlify

Reply:
Yep noticed that too and removed it.